Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -464,6 +521,8 @@ extension PaymentSheetTestPlayground {
             self.clientSecret = json["intentClientSecret"]
             self.ephemeralKey = json["customerEphemeralKeySecret"]
             self.customerID = json["customerId"]
+            self.paymentMethodTypes = json["paymentMethodTypes"]?.components(separatedBy: ",")
```

Comment:
Left this as a string for easier decoding, also didn't want to possibly break Android  or master by changing our server from returning [String: String] to [String: Any]